### ISSUE TITLE
ConnectTimeout option is not tend to work properly

### ISSUE THREAD

**Issue by Mixser** [nikepan/clickhouse-bulk#28]
Hi, 

At `NewClickhouse` method your have been implemented not right behavior for the `ConnectTimeout` options:

```go
c.ConnectTimeout = connectTimeout
if c.ConnectTimeout > 0 {
    c.ConnectTimeout = 10
}
```

So, if I will set any positive value for `ConnectTimeout` - it will be not used but rewritten to `10 seconds`;

**Comment by nikepan** [comment]
Oh, thanks!

**Comment by nikepan** [comment]
I fixed it in release v1.2.6
